packages/expo-maps/ios/AppleMapsModule.swift

```diff
@@ -17,6 +17,10 @@ public class AppleMapsModule: Module {
       AsyncFunction("setCameraPosition") { (view: AppleMapsViewWrapper, config: CameraPosition?) in
         view.setCameraPosition(config: config)
       }
+
+      AsyncFunction("openLookAroundAsync") { (view: AppleMapsViewWrapper, coordinates: Coordinate, promise: Promise) in
+        view.openLookAround(coordinate: coordinates, promise: promise)
+      }
```

Comment:
```suggestion
      AsyncFunction("openLookAroundAsync") { (view: AppleMapsViewWrapper, coordinates: Coordinate) in
        try await view.openLookAround(coordinate: coordinates)
      }
```

Reply:
@alanjhughes I tried changing the function signature like this, but now the code doesnt even reach the body of this function. Does something need to be changed on the JS side as well? if I keep the await on the JS side it complains that I haven't provided enough arguments, and if I remove the await on the JS side then it acts like nothing happened. Any ideas?

Reply:
Seems to be a problem with our `ConcurrentFunctionDefinition`. For now you can do this
```swift
 AsyncFunction("openLookAroundAsync") { (view: AppleMapsViewWrapper, coordinates: Coordinate) in
    Task {
      try await view.openLookAround(coordinate: coordinates)
    }
  }
```